app/assets/javascripts/student_profile/ServicesList.js

```diff
@@ -146,26 +138,27 @@ export default class ServicesList extends React.Component {
     const wasDiscontinued = this.wasDiscontinued(service);
     const momentStarted = moment.utc(service.date_started);
 
+    // For discontinued services, display the length of time between start and discontinue dates
     if (wasDiscontinued) {
-      // For discontinued services, display the length of time between start and discontinue dates
       return (
         <div>
           {moment.utc(service.discontinued_recorded_at).from(moment.utc(service.date_started), true)}
         </div>
       );
-    } else {
-      const startedToday = moment().utc().subtract(1, 'day') < momentStarted;
+    }
 
-      // Don't show how long service has been going if it was added today
-      if (startedToday) return null;
+    // Don't show how long service has been going if it was added today
+    const {nowFn} = this.context;
+    const nowMoment = nowFn();
```

Comment:
This is the substantive change - changing from `moment.utc()` to using the time provided by context.